lib/broadway/acknowledger.ex

```diff
@@ -38,6 +38,19 @@ defmodule Broadway.Acknowledger do
   @callback ack(ack_ref :: {pid, reference}, successful :: [Message.t()], failed :: [Message.t()]) ::
               :ok
 
+  @doc """
+  Configures the acknowledger with new `options`.
+
+  Every acknowledger can decide how to incorporate the given `options` into its
+  `ack_data`. The `ack_data` is the current acknowledger's data. The return value
+  of this function is `{:ok, new_ack_data}` where `new_ack_data` is the updated
+  data for the acknowledger.
+  """
+  @callback configure(ack_ref :: {pid, reference}, ack_data :: term, options :: keyword) ::
```

Comment:
ack_ref should be a term! :D Can you please change this module accordingly?

Reply:
Let's make it clear in the docs that `options` is whatever the acknowledger chooses to support!

Reply:
Done and done :)